options/linux/generic/mntent-stubs.cpp

```diff
@@ -1,16 +1,23 @@
 
+#include <errno.h>
 #include <mntent.h>
 #include <stdio.h>
+#include <limits.h>
 #include <string.h>
 #include <bits/ensure.h>
 
+static char *internal_buf;
+static size_t internal_bufsize;
+
+#define SENTINEL (char *)&internal_buf
```

Comment:
I am not a fan of these being global. is there no cleaner way to achieve the same or similar (for example passing in null to `getmntent_r` and checking against that instead)

Reply:
I have placed `internal_buf(size)` in an anonymous namespace.